src/component/RoadmapCanvas.js

```diff
@@ -207,6 +225,49 @@ export class RoadmapCanvas extends EventEmitter {
         }
       })
     );
+    this.yAxis
+      .attr('x', marginLeft);
+  }
+
+  /**
+   * @param {number} h
+   * @private
+   */
+  _updateBackground(h) {
+    let colorMap = [
+      '#f9f9f9',
+      '#eeeeee'
+    ];
+    const gap = h / this.yAxisMap.length;
```

Comment:
I can't get the accurate step.
Maybe, it has an error (but so small error) when bar height is calculated.